.gitignore

```diff
@@ -10,3 +10,4 @@ compiled
 .awcache
 .rpt2_cache
 docs
+test/auth.json
```

Comment:
No test files were added to the project. Should this be removed, or did you forget to include the other files?

If you want to create a basic bot for testing, it will just need a main file, and a commands folder with a few basic commands to test the different features, like testing the args, testing error handling, etc.

Reply:
I added at least one command, will add more in a later PR if necessary.